content/departments/engineering/teams/integrations/integrations-support-rotation.md

```diff
@@ -0,0 +1,58 @@
+# Integrations support rotation
+
+> Note: This processes is currently in trial mode, meaning we will re-evaulate the value it brings on Nov 21 (~2 months after the introduction).
+
+The [Ingrations team](./index.md) has a dedicated member on a weekly support rotation. This document outlines the responsibilities of this role and how to execute the tasks. In a nutshell, these are the responsibilities of the engineer:
+
+- Respond and triage incoming inquiries from other teams and the support channels
+
+## Expectations
+
+The engineer on rotation is expected to comply with these responsibilities only **_within their usual working hours_**. There is no on-call, 24x7-watch expectation on these tasks. The async nature of Sourcegraph allows us to properly support the organization without having to be in front of the computer all day.
```

Comment:
Is the expectation that this person's sole responsibility is the support rotation? This would mean we should expect them not to do any other work during the week they are on rotation?

Reply:
No. In fact I don't anticipate that this rotation is going to need more than 10% of the person's time at all. I'll clarify this.

Reply:
I’m wondering which wording let to your confusion?

I’m thinking of adding this:

> We do not expect work related to the support rotation to take up more than 10% of your time.

Reply:
I don't think it was any explicit wording, I think this was a bias from how I've seen this work in the past.

I think that's a good addition :)